redaxo/src/addons/media_manager/boot.php

```diff
@@ -15,4 +15,35 @@
     // delete thumbnails on mediapool changes
     rex_extension::register('MEDIA_UPDATED', [rex_media_manager::class, 'mediaUpdated']);
     rex_extension::register('MEDIA_DELETED', [rex_media_manager::class, 'mediaUpdated']);
+   	rex_extension::register('MEDIA_IS_IN_USE', 'rex_media_manager_media_is_in_use');
```

Comment:
```suggestion
    rex_extension::register('MEDIA_IS_IN_USE', [rex_media_manager::class, 'mediaIsInUse']);
```

Und dann entsprechende die Funktion dahin verschieben.

Reply:
Gute Idee. Ich war mir nicht sicher wo die Funktionalität besser aufgehoben ist.